charts/umbrella/values-adopter-irs.yaml

```diff
@@ -0,0 +1,87 @@
+# #############################################################################
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# #############################################################################
+---
+
+centralidp:
+  enabled: true
+
+discoveryfinder:
+  enabled: true
+  discoveryfinder:
+    authentication: false
+    properties:
+      discoveryfinder:
+        initialEndpoints:
+          - type: bpn
+            endpointAddress: http://umbrella-discovery-mock:8080/api/administration/connectors/discovery
+            description: Service to discover connector endpoints based on bpns
+            documentation: http://portal-backend.tx.test/api/administration/swagger/index.html
+
+semantic-hub:
+  enabled: false
+  graphdb:
```

Comment:
you can remove this line because the chart is enabled.

Reply:
thanks, I removed it